shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API.hs

```diff
@@ -46,6 +47,6 @@ class
     ChainData (State (Core.EraRule "PPUP" era)),
     SerialisableData (State (Core.EraRule "PPUP" era))
   ) =>
-  ShelleyBasedEra era
+  ShelleyBasedEra era utxoenv
```

Comment:
I don't think this is going to work. Consensus uses this to cover all shelley based ledgers, but this parameter will have to vary.

Reply:
ok, that's what I was worried about, thanks!